axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/TeamTaskBusinessService.java

```diff
@@ -25,9 +25,15 @@
 import com.axelor.team.db.TeamTask;
 import java.math.BigDecimal;
 import java.time.LocalDateTime;
+import java.util.Map;
 
 public interface TeamTaskBusinessService extends TeamTaskService {
+
   TeamTask create(SaleOrderLine saleOrderLine, Project project, User assignedTo);
 
   TeamTask create(TaskTemplate template, Project project, LocalDateTime date, BigDecimal qty);
+
+  Map<String, Object> getDiscount(TeamTask teamTask, BigDecimal unitPrice);
```

Comment:
Method should be 'updateDiscount' and it should only take 'teamTask' as parameter as unitPrice comes from TeamTask. and return type should be 'TeamTask' with updated discount values.